docs/scalardb-server.md

```diff
@@ -0,0 +1,38 @@
+# Scalar DB server
+
+This document explains the Scalar DB server that is a gRPC interface of Scalar DB.
```

Comment:
```suggestion
Scalar DB server is a gRPC server that implements Scalar DB interface. 
With Scalar DB server, you can use Scalar DB features from multiple programming languages that are supported by gRPC.
This document explains how to install and use Scalar DB server.
```

Reply:
As we use the gRPC bidirectional stream in the Scalar DB server, I think it isn't straightforward to implement client code. So, I didn't think we should mention that the Scalar DB server can support multiple programming languages via gRPC. What do you think?

Reply:
I think we can mention that for other language support, the client sample will be provided in the future ( in x.x.x release for example)?

Reply:
I think we should mention but we should note that it is difficult since it's using the bi-directional stream of gRPC and you need to be familiar with it to properly implement a client. We can also mention that we will provide sample code or official client SDKs in the future.

Otherwise, I think users/readers would think what the server is for.

Reply:
Thank you for the suggestions! I will revise it.